codice/alliance
# CAL-422 Updated ImageAttribute class

* added image comments 4-9 to ImageAttribute
* updated ImageAttributeTest
* updated Nitf_Segment_Image-table.adoc

#### What does this PR do?
Updates ImageAttribute class to support image comments 4-9.
#### Who is reviewing it? 
@harrison-tarr 
@roelens8 
@jaymcnallie 
#### Choose 2 committers to review/merge the PR.
@jaymcnallie
@shaundmorris
#### How should this be tested?
1. ingest a nitf that contains comments in segments 4 and 9
2. open Intrigue and search for the image
3. verify the metadata for the image has extracted all the comments in segments 4 and 9
#### Any background context you want to provide?
IAW MIL-STD-2500C the image comments can have 0-9 blocks.  The ImageAttribute class only exposes the first three blocks.
#### What are the relevant tickets?
[CAL-422](https://codice.atlassian.net/browse/CAL-422)
#### Screenshots (if appropriate)
#### Checklist:
